hssqlppp/tests/Database/HsSqlPpp/Tests/Parsing/CreateTable.lhs

```diff
@@ -350,8 +350,110 @@ quick sanity check
 >       [createTable "t" [att "a" "int"]
 >        `setTableOpts` [TableOptionNumberVal ["yes"] "5.5"]]-}
 >      ]
-
+>    ,Group "External tables" [
+>       s "create external table test (\n\
```

Comment:
Please use `Stmt` instead of `s` and don't add these newlines and `\` at the end of lines.